### PR TITLE
Feature #103 - Payment Form validation

### PR DESCRIPTION
**Pull request by abefernan**
**Description**

This PR introduces validation in the send payment form and hardcoded values on the dropdown as suggested by Issue #103.

### CODE REVIEW

File: packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx [hunk 1]
```diff
@@ -38,16 +96,26 @@ export const CurrencyToSend = () => {
...
+            <TextField
+              placeholder="0,00"
+              fullWidth
+              onBlur={textFieldHandleBlur}
+              onChange={textFieldHandleChange}
+            />
           </Block>
-          <Select />
+          {/*TODO properly style Select component*/}
+          <Select value={currency} onChange={selectHandleChange} autoWidth>
```
Comment (apanizo) [inline packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx]: There is a SelectField in `medulas-react-components` you can use.

File: packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx [hunk 2]
```diff
@@ -38,16 +96,26 @@ export const CurrencyToSend = () => {
         </Typography>
         <Block display="flex" marginTop={5} marginBottom={1}>
           <Block marginRight={1}>
-            <TextField placeholder="0,00" />
+            <TextField
```
Comment (apanizo) [inline packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx]: Use the TextField from medulas here as well, and delete the onBlur stuff.

File: packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx [hunk 3]
```diff
@@ -38,16 +96,26 @@ export const CurrencyToSend = () => {
         </Typography>
         <Block display="flex" marginTop={5} marginBottom={1}>
```
Comment (apanizo) [inline packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx]: You should wrap all fields elements in a Form, which is also available in medulas, doing this, you should implement validation in the same way as we are doing, in the same package as in medulas you can see a story, or directly find forms in the extension.

File: packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx [hunk 4]
```diff
@@ -20,13 +24,86 @@ const useStyles = makeStyles((theme: Theme) => ({
...
+
+const CurrencyToSend = () => {
   const classes = useStyles();
 
   const avatarClasses = {
     root: classes.avatar,
   };
 
+  //NOTE hardcoded initial state for "currency"
+  const [currency, setCurrency] = useState(currencyItems[1].name);
```
Comment (apanizo) [inline packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx]: Please put all hooks once after other, meaning move down the avatarClasses. In this way is easier for the eye and harder to introduce bugs.

**Comment by apanizo** [comment]
Do not forget to rebase! 😉

File: packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx (hunk 4 above)
Comment (abefernan) [reply packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx]: Fixed by [99da2af](https://github.com/iov-one/ponferrada/pull/124/commits/99da2af71c58070eed2dffe202fc5b44feb007da).

**Pull request merged**